R/aggregate_kicking_stats.R

```diff
@@ -0,0 +1,161 @@
+#' Summarize Kicking Stats
```

Comment:
We recently added defensive player stats and I think this file should be named aggregate_game_stats_kicking.R to be consistent with that

Reply:
Good eye--updated the file name!